src/App.scss

```diff
@@ -90,7 +90,7 @@ body {
 }
 
 .container {
```

Comment:
Currently this .container definition overrides all react-bootstrap container components. I've changed the margin-top to 0px to correctly display on the home page but as a side effect all flexboxes on other pages are changed.

Reply:
I haven't done a lot of react, is this just because it is the App.scss file? or would any Page.scss also be applied globally? (aka are we good to just make Page.scss for each page and override stuff for specific pages?)

Reply:
Yep, app.scss is imported in app.jsx, so every component inherits these styles. React-bootstrap's container is becomes <div class='container'>.